package.json

```
@@ -75,68 +76,48 @@
     "react-native-storybook-loader": "^2.0.4",
     "react-native-swiper": "^1.6.0",
     "react-native-tab-view": "^2.15.2",
-    "react-native-vector-icons": "^7.1.0",
+    "react-native-vector-icons": "^8.1.0",
     "react-native-web": "^0.14.10",
     "react-native-webview": "^11.0.2",
     "react-native-youtube-iframe": "^1.4.0",
+    "react-query": "^3.19.0",
     "rn-fetch-blob": "^0.12.0",
     "styled-components": "^5.2.1",
     "validator": "^13.5.2",
-    "yarn": "^1.22.10"
+    "yarn": "^1.22.17"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.9",
-    "@babel/runtime": "^7.12.5",
-    "@react-native-community/eslint-config": "^2.0.0",
     "@types/eslint": "^7.2.13",
     "@types/jest": "^27.0.2",
     "@types/json-stable-stringify": "^1.0.33",
+    "@babel/core": "^7.14.6",
+    "@babel/runtime": "^7.14.6",
+    "@react-native-community/eslint-config": "^3.0.0",
     "@types/react": "^16.13.1",
     "@types/react-native": "^0.63.37",
     "@types/react-test-renderer": "^16.9.4",
     "@types/styled-components": "^5.1.14",
-    "@types/styled-components-react-native": "^5.1.1",
-    "@typescript-eslint/eslint-plugin": "^4.27.0",
-    "@typescript-eslint/parser": "^4.27.0",
-    "babel-jest": "^25.1.0",
-    "eslint": "^7.32.0",
+    "babel-jest": "^27.0.6",
+    "eslint": "^7.31.0",
     "eslint-config-airbnb": "^18.2.1",
     "eslint-config-prettier": "^8.3.0",
     "eslint-import-resolver-babel-module": "^5.3.1",
     "eslint-plugin-import": "^2.24.2",
-    "eslint-plugin-jest": "^24.4.2",
     "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.4.0",
     "eslint-plugin-react": "^7.26.0",
-    "jest": "^27.2.3",
-    "prettier": "^2.4.1",
-    "react-native-clean-project": "^3.6.3",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "jest": "^25.1.0",
+    "prettier": "^2.3.2",
+    "react-native-clean-project": "^3.6.4",
     "react-test-renderer": "16.13.1",
```

This one should match the version of `react`, so bump to 17.0.2 **only if** you also bump `react` version to 17. Otherwise let it be.
```suggestion
    "react-test-renderer": "17.0.2",
```

---

Done!